src/pudl/metadata/fields.py

```diff
@@ -2329,6 +2377,7 @@
         "description": "Original reported energy interchange between adjacent balancing authorities.",
         "unit": "MWh",
     },
+    "irs_number": {"type": "string", "description": "ID of the company with the IRS."},
```

Comment:
In our current naming conventions we use `_id` to identify numerical IDs, and put the data source at the end of the field name, so I think this would probably be `company_id_irs`.

Reply:
Changed.